Python tutorial not correct

In this tutorial https://docs.opencv.org/master/d0/d86/tutorial_py_image_arithmetics.html (-> bitwise operation.) image sizes are not correct.

 I have got an error :
cv2.error: C:\Lib\opencv\modules\core\src\arithm.cpp:241: error: (-215) (mtype == 0 || mtype == 1) && _mask.sameSize(*psrc1) in function cv::binary_op

if I write :

```
img2 = cv.imread('c:/lib/opencv/samples/data/opencv-logo.png')
img2=cv.resize(img2,None,fx=0.25,fy=0.25)
```

there is no error but I don't understand how can i get this [image](https://docs.opencv.org/master/overlay.jpg) with add function (with copyto and a mask I can get this result).

--- a/doc/py_tutorials/py_core/py_image_arithmetics/py_image_arithmetics.markdown
+++ b/doc/py_tutorials/py_core/py_image_arithmetics/py_image_arithmetics.markdown
@@ -77,7 +77,7 @@ bitwise operations as below:
 @code{.py}
 # Load two images
 img1 = cv.imread('messi5.jpg')
-img2 = cv.imread('opencv-logo.png')
+img2 = cv.imread('opencv-logo-white.png')
 
 # I want to put logo on top-left corner, So I create a ROI
 rows,cols,channels = img2.shape

